distribution/src/main/release/samples/pom.xml

```diff
@@ -210,6 +216,49 @@
             </plugins>
         </pluginManagement>
     </build>
+    <repositories>
+        <repository>
```

Comment:
I think we could remove snapshots, it was temporary before the milestones became available

Reply:
Done.